pkg/ceph/osd/device.go

```diff
@@ -50,6 +50,9 @@ const (
 	key = %s
 	caps mon = "allow profile bootstrap-osd"
 `
+	// ratio of disk space that will be used by bluestore on a dir.  This is an upper bound and it
+	// is not preallocated (it is thinly provisioned).
+	bluestoreDirBlockSizeRatio = 0.5
```

Comment:
PUTs will fail after the bluestore dir exceeds that ratio of usage?

Reply:
Basically, yes.  Writes will fail once the actual block store gets to 50% of the disk.  Now that I think about it, that's probably way too low.  I imagine that the main use cases for OSDs are most always intending to use the storage.  I think I'll bump this up to 90%.